test/common/index.js

```diff
@@ -255,7 +255,7 @@ function platformTimeout(ms) {
 }
 
 let knownGlobals = [
-  AbortController,
+//  AbortController,
```

Comment:
I don’t think you wanted to commit this, although I agree that this can be quite annoying sometimes and we might want to try to come up with a better solution here (e.g. specifying the global property names instead of the values?)

Reply:
ah, lol, good catch :-)